src/Responses/FailureResponse.php

```diff
@@ -23,7 +23,7 @@ public function __construct($headers, $body, $http_code, callable $failure_callb
         parent::__construct($headers, $body, $http_code);
 
         if ($failure_callback) {
-            call_user_func($failure_callback);
+            call_user_func($failure_callback, $this);
```

Comment:
Would this mean the first argument for the signature of all callbacks would need expect an instance of `MailchimpResponse`? That would likely be a breaking change for other implementers.

Reply:
Seems that php throws `ArgumentCountError('Too few arguments to function..')` only if function expects more arguments than were passed. 
In case we pass more arguments than expected, it will continue work.
Just tested this with installed PHP 7.3.9

Reply:
Or maybe just start new branch 3.2 if possible

Reply:
I think that would be good.